translate/storage/test_mo.py

```diff
@@ -156,17 +155,14 @@ def test_output(self):
                 # can skip the checks here.
                 continue
 
-            mo_msgfmt_f = open(MO_MSGFMT)
-            mo_pocompile_f = open(MO_POCOMPILE)
-
-            try:
+            with open(MO_MSGFMT, 'rb') as mo_msgfmt_f:
                 mo_msgfmt = mo_msgfmt_f.read()
-                print("msgfmt output:")
-                print(repr(mo_msgfmt))
+            print("msgfmt output:")
+            print(repr(mo_msgfmt))
+
+            with open(MO_POCOMPILE, 'rb') as mo_pocompile_f:
                 mo_pocompile = mo_pocompile_f.read()
-                print("pocompile output:")
-                print(repr(mo_pocompile))
-                assert mo_msgfmt == mo_pocompile
-            finally:
-                mo_msgfmt_f.close()
-                mo_pocompile_f.close()
+            print("pocompile output:")
+            print(repr(mo_pocompile))
+
+            assert mo_msgfmt == mo_pocompile
```

Comment:
Please note that this must be only run if the two files have been successfully opened and read.